### PR TITLE
Implement methods for the MusicAlbum class

### LINKED ISSUE #21: [2pt] Implement methods in the MusicAlbum class
- `add_item` method in the `Genre` class:
  - Should take an instance of the Item class as an input.
  - Should add the input item to the collection of items.
  - Should add self as a property of the item object (by using the correct setter from the item object).

- `can_be_archived?()` in the `MusicAlbum` class:
  - Should override the method from the parent class.
  - Should return `true` if parent's method return true AND if `on_spotify` equals `true`
  - Otherwise, it should return `false`.

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:
- Created the `genre.rb` file introducing the `Genre` class.
- Added a setter for the `genre` variable in the `MusicAlbum` class.
- Created the `add_item()` method for the `Genre` class.

closes #21

### CODE REVIEW

File: music_album.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class MusicAlbum < Item
   DEFAULT_BOOL = false
 
-  attr_accessor :publish_date
+  attr_accessor :genre, :publish_date
 
   def initialize(genre, author, label, publish_date, on_spotify = DEFAULT_BOOL)
```
Comment (medaminedev66) [inline music_album.rb]: Hi Ruben, You have done a great job. I just want to let you know that we have changed the parameters of initialize method in the parent class `Item`. I think it should be changed in The child class `MusicAlbum` too.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]

File: music_album.rb (hunk 1 above)
Comment (rpire) [reply music_album.rb]: Hi @medaminedev66

Already working on those changes. 👍

**Comment by medaminedev66** [comment]
Thanks, @rpire for implementing the required changes. Your tasks are complete! There is nothing else to say other than... it's time to merge it

**Review by medaminedev66 — APPROVED** [review]

**Comment by rpire** [comment]
Hi @medaminedev66 👋

Thank you for reviewing my work. I will proceed to merge these changes into the `dev` branch. 🚀

Cheers! 🤓

**Pull request merged**